docs/opcodes/55.mdx

```diff
@@ -37,5 +37,5 @@ group: Stack Memory Storage and Flow Operations
 The state changes done by the current context are [reverted](#FD) in those cases:
 - Not enough gas.
 - Not enough values on the stack.
-- The current execution context is from a [STATICCALL](/#FA) (since Byzantium fork).
+- The current execution context is from a [STATICCALL](/#FA) (since Byzantium fork) or an [EXTSTATICCALL](/#FB) (since Prague fork).
```

Comment:
This is the only change I'm a bit torn about, we mainly want to just cover the EOF changes and not really reference `Prague` yet. 

How do you feel about just calling it `EOF fork` instead ? As that is how this new requirement gets introduced.

*(it's been added a few places, same comment there, I only comment on this case to as I think you can search for the rest)*